internal/server/storage/drivers/generic_vfs.go

```diff
@@ -332,6 +333,36 @@ func genericVFSCreateVolumeFromMigration(d Driver, initVolume func(vol Volume) (
 
 		defer func() { _ = to.Close() }()
 
+		st, err := to.Stat()
```

Comment:
These two blocks are identical, right?  Any reason not to use a common helper?  Seems like future you will want that.

Reply:
@hallyn fair point, updated :)